roles/pbx/tasks/freepbx.yml

```diff
@@ -142,7 +147,9 @@
     collation: utf8_general_ci
     login_host: "{{ asterisk_db_host }}"
```

Comment:
This line is a duplicate of...2 lines down.

```suggestion
```

Reply:
addressed